pkg/proxy/ipvs/proxier.go

```diff
@@ -1144,8 +1151,16 @@ func (proxier *Proxier) syncProxyRules() {
 		// proxier.kubeServiceAccessSet.activeEntries.Insert(entry.String())
 		// Install masquerade rules if 'masqueradeAll' or 'clusterCIDR' is specified.
 		if proxier.masqueradeAll {
+			if valid := proxier.clusterIPSet.validateEntry(entry); !valid {
```

Comment:
This code is identical in both cases, move it up above the if/else conditional.

Reply:
Done